CHANGELOG.asciidoc

```diff
@@ -31,6 +31,8 @@ https://github.com/elastic/beats/compare/v5.0.0-alpha5...master[Check the HEAD d
 ==== Bugfixes
 
 *Affecting all Beats*
+- Use indentation when generating the index pattern {pull}2217[2217]
+- Generate the kibana type of each index pattern field from the field type, used in the template {pull}2220[2220]
```

Comment:
yes, both are bug fixes as Kibana complains after you import the index patterns in Kibana.